index.html

```diff
@@ -63,80 +70,69 @@ <h1 class="header-text">Hey there. I’m Raphael
-                    <ul class="skills"><li>Ruby on rails</li><li>CSS</li><li>JavScript</li><li>html</li></ul>
-                    <div class="button"><p>See Project</p></div>
-                </div>    
             </div>
-
-            <div class="card-item">
-               
-                <div class="down-card">
-                    <h3>Multi-Post Stories Gain+Glory</h3>
-                    <ul class="skills"><li>Ruby on rails</li><li>CSS</li><li>JavScript</li><li>html</li></ul>
-                    <div class="button"><p>See Project</p></div>
-                </div>    
+            <!-- pop up  -->
+            <div class="pop-up-layer" id="pop-up-layer">
+                <div class="main-pop-up">
+                    <div class="close-pop" onclick="closePopUp()">
+                        <p>X</p>
+                    </div>
+                    <div class="pop-img"><img src="img/Snapshoot Portfolio.png" alt="recent work" width="20%"></div>
+                    <div class="pop-window">
+                        <div class="pop-head-text">
+                            <h3>Keeping track of hundreds of
+                                components</h3>
+                        </div>
+                        <div class="pop-skills-display">
+                            <ul>
+                                <li>Codekit</li>
+                                <li>GitHub</li>
+                                <li>JavScript</li>
+                                <li class="hide-pop">Bootstrap</li>
+                                <li class="hide-pop">Terminal</li>
+                                <li class="hide-pop">Codepen</li>
+                            </ul>
+                        </div>
+                        <div class="pop-d">
+                            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                                Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
+                                when an unknown printer took a galley of type and scrambled it 1960s with the
+                                releaLorem Ipsum is simply dummy text of the printing and typesetting industry.
+                                Lorem Ipsum has been the industry's
+                                standard dummy text ever since the 1500s,
+                                when an unknown printer took a galley of type and scrambled it 1960s with the relea</p>
+                        </div>
+                        <div class="pop-buttons">
+                            <button type="submit" class="see-live">See Live <img src="img/seeliveicon.svg"
+                                    alt="see live icon"></button>
+                            <button type="submit" class="see-source">See Source <img src="img/Vector.png"
+                                    alt="see Source icon"></button>
+                        </div>
+                    </div>
+                </div>
```

Comment:
- [ ] Good job generating the projects section dynamically using javascript. But the popup is hardcoded, I am seeing the same popup for all projects. Kindly utilize the data stored in the array of objects to generate your popup dynamically. The data in the array of objects should be reflected in the popup. ✍️
Check the example below to structure your array of objects.

```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```